media/js/firefox/family/copy-to-clipboard.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// if the user clicks on c-module-tag, copy link to clipboard
+
+(function () {
+    'use strict';
+
+    // var copyText = document.querySelector('.c-module-copy');
+    // var copiedText = document.querySelector('.c-module-copied');
+
+    var linksArray = [];
+
+    document.querySelectorAll('.c-module-tag').forEach(function (occ) {
+        linksArray.push(occ);
+    });
+
+    linksArray.map(function (e) {
+        e.addEventListener('click', function copyLink() {
+            var link = e.href;
```

Comment:
One last though, is it worth calling `preventDefault()` here on click, to prevent the page jumping (which makes the tooltip a bit difficult to read?)

```suggestion
        e.addEventListener('click', function (e) {
            e.preventDefault();
            var link = e.href;
```

The downside to this is it stops the URL from changing to include the hash, but it does make the copy experience a bit less jarring.

Not entirely sure which is the best to be honest, just calling it out.

Reply:
Do you mean we should use it to prevent the page from scrolling to the top of the section that was clicked?

Reply:
Right, I click copy and the page jumps up, which makes the notification pretty hard to read